invenio_communities/webpack.py

```diff
@@ -54,6 +54,8 @@
                 '@ckeditor/ckeditor5-react': '^2.1.0',
                 '@semantic-ui-react/css-patch': '^1.0.0',
                 'react-invenio-forms': '^0.10.0',
+                # TODO update after releasing Funding Field
+                'react-invenio-deposit': '^0.19.4',
```

Comment:
This should be setup to the correct version after https://github.com/inveniosoftware/react-invenio-deposit/pull/465/files gets released